PendingReleaseNotes

```diff
@@ -77,7 +77,10 @@
   map and unmap images in namespaces using the `image-spec` syntax since then
   but the corresponding option available in most other commands was missing.
 * RGW: Compression is now supported for objects uploaded with Server-Side Encryption.
-  When both are enabled, compression is applied before encryption.
+  When both are enabled, compression is applied before encryption. Note that earlier
+  releases of multisite do not replicate such objects correctly, so all zones must
+  upgrade to Reef before enabling the `compress-encrypted` zonegroup feature: see
```

Comment:
_Reef or  later_ ?

Reply:
this is for the Reef release notes, so there is no 'or later' in this context. but i did adjust the documentation for the 'compress-encrypted' feature to specify 'Reef or later'. thanks